### DISCUSSION TITLE
Filter issues by participant / comment author

### DISCUSSION THREAD

**Discussion by donquixote** [community/community#45354] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

Currently we can filter issues by author. This makes it possible to show only issues I created.

I would like to be able to filter by "participant": Show every issue where I am either the author of the original post, _or_ I posted a comment. (TBD if we also count smilies and upvotes)

This is mostly relevant for participant = myself, but it can also be useful to filter by other users.

**Comment by donquixote** [comment]
I see now there is `ìnvolves:USERNAME` in advanced search syntax.
Unfortunately it is not in the UI, so most people will never know about it.
Perhaps there could be a dropdown with more filter options?

**Reply by Patrik-Br** [reply]
Thank you so much, I was looking for this!

**Reply by Ten0** [reply]
I made the mistake to copy/paste "involves" from the post above and took some time to figure out why that didn't work. The `ì` is a trap. 😅

**Comment by sanjarcode** [comment]
This is needed so badly, especially when searching for comments that were made by someone (maybe myself) a long time ago.

- From a technical point of view, maybe this is missing because it's easier to search issues created by a user, or mentioned - O(n). Involves has to search all comment boxes of all issues (n<sup>2</sup>), which may be slow. Still, this is quite important.
- If this is missing, then recent comments made by a user (in any repo) - i.e. history of all comments you made, might also be missing.

**Comment by Nicksisw** [comment]
Good

**Comment by toransahu** [comment]
+1

**Comment by StephanBijzitter** [comment]
I was looking for the same filter, but for a slightly different reason:

I want to see issues where I've been a participant (creator/assignee/commenter/tagged in a comment) and sort most recently updated first

**Reply by clover-yan** [reply]
```
sort:updated-desc is:issue involves:@me 
```
